frontend/src/metabase/static-viz/components/LineAreaBarChart/LineAreaBarChart.tsx

```diff
@@ -53,21 +51,24 @@ const LineAreaBarChart = ({
 
   const minTickSize = chartStyle.axes.ticks.fontSize * 1.5;
   const xValuesCount = getXValuesCount(series);
-  const chartSize = calculateChartSize(settings, xValuesCount, minTickSize);
-  const adjustedSettings = adjustSettings(
+  const { width, height } = calculateChartSize(
     settings,
     xValuesCount,
     minTickSize,
-    chartSize,
+  );
+  const adjustedSettings = useMemo(
+    () =>
+      adjustSettings(settings, xValuesCount, minTickSize, { width, height }),
```

Comment:
I destructure `chartSize` since its changing every render, so it couldn't be directly passed to `adjustSettings` without being memoized, this is a lazy way to avoid having to memoize `chartSize` since it's just a map of numbers.

Reply:
@WiNloSt I believe the actual static viz rendering environment does not have any rerenders, wdyt? Anyway, there is no harm from useMemo there

Reply:
You are absolutely correctly. I can remove `useMemo` here to simplify the code.